Source/WebKit/Shared/glib/ArgumentCodersGLib.cpp

```diff
@@ -162,4 +164,42 @@ std::optional<GRefPtr<GTlsCertificate>> ArgumentCoder<GRefPtr<GTlsCertificate>>:
 }
 template std::optional<GRefPtr<GTlsCertificate>> ArgumentCoder<GRefPtr<GTlsCertificate>>::decode<Decoder>(Decoder&);
 
+void ArgumentCoder<GRefPtr<GUnixFDList>>::encode(Encoder& encoder, const GRefPtr<GUnixFDList>& fdList)
+{
+    if (!fdList) {
+        encoder << false;
+        return;
+    }
+
+    Vector<UnixFileDescriptor> attachments;
+    unsigned length = std::max(0, g_unix_fd_list_get_length(fdList.get()));
+    if (!!length) {
```

Comment:
I don't get it... there's no need to convert to bool here, is there? Why not just write `if (length)`?

Reply:
There's conversion to a boolean value either way. This just makes it explicit through the double negation. I like to use this for general clarity.